From 50caebab484332838a1fba5ebeccd8b9b5545762 Mon Sep 17 00:00:00 2001
From: Maurizio Lombardi <mlombard@redhat.com>
Date: Mon, 8 Sep 2014 13:44:04 -0400
Subject: [scsi] bnx2i: Make boot_nic entry visible in the sysfs session objects

Message-id: <1410183844-20977-1-git-send-email-mlombard@redhat.com>
Patchwork-id: 92004
O-Subject: [RHEL7.1 e-stor PATCH] bnx2i: Make boot_nic entry visible in the sysfs session objects
Bugzilla: 1139139
RH-Acked-by: Ewan Milne <emilne@redhat.com>
RH-Acked-by: Rob Evers <revers@redhat.com>

posted on linux-scsi mailing list and ACKed by Eddie Wai

Signed-off-by: Tej Parkash <tej.parkash@qlogic.com>
Signed-off-by: Vikas Chaudhary <vikas.chaudhary@qlogic.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/bnx2i/bnx2i_iscsi.c b/drivers/scsi/bnx2i/bnx2i_iscsi.c
index 8bb9cac..7dd9e9b 100644
--- a/drivers/scsi/bnx2i/bnx2i_iscsi.c
+++ b/drivers/scsi/bnx2i/bnx2i_iscsi.c
@@ -2235,6 +2235,9 @@ static umode_t bnx2i_attr_is_visible(int param_type, int param)
   case ISCSI_PARAM_TGT_RESET_TMO:
   case ISCSI_PARAM_IFACE_NAME:
   case ISCSI_PARAM_INITIATOR_NAME:
+  case ISCSI_PARAM_BOOT_ROOT:
+  case ISCSI_PARAM_BOOT_NIC:
+  case ISCSI_PARAM_BOOT_TARGET:
    return S_IRUGO;
   default:
    return 0;
-- 
1.7.1